cachito/web/static/api_v1.yaml

```diff
@@ -277,6 +277,75 @@ paths:
                 $ref: "#/components/schemas/RequestConfiguration"
       security:
       - "Kerberos Authentication": []
+  "/requests/{id}/content-manifest":
+    get:
+      description: Return the content manifest of a request
+      parameters:
+      - name: id
+        in: path
+        required: true
+        description: The ID of the Cachito request to retrieve the content manifest for
+        schema:
+          type: integer
+      responses:
+        "200":
+          description: The content manifest of the Cachito request
+          content:
+            application/json:
+              schema:
+                type: object
+                nullable: true
+                properties:
+                  metadata:
+                    type: object
+                    properties:
+                      icm_version:
+                        type: integer
+                        example: 1
+                      icm_spec:
+                        type: string
+                        example: "https://json-schema.icm.example.com"
+                      image_layer_index:
+                        type: integer
+                        example: 1
```

Comment:
Will Cachito ever return an integer other than `-1` since it has no context of the image that the request is used for?

Reply:
Changed to `-1`